### PR TITLE
Added a file which states the properties and method of arrays

### LINKED ISSUE #77: I will improve an existing tutorial of array with adding some properties or methods of it.
**Description**
I will improve an existing tutorial of array with adding some properties or methods of it.

**Do you want to be assigned to work on this?**
(mark [x])
- [x] YES
- [ ] NO

**Optional: Any additional info you want to add**
I've read the CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rishabh1911007**
### 2. Description
I added a file in folder (3-arrays) which states the properties method of arrays

### 3. Fill in checklist by marking [x]

- [x] I've read the CONTRIBUTING.md
- [x] I was assigned to this issue
- [x] My code is formatted using required linting
- [x] I've run my code locally and checked it works

### CODE REVIEW

File: 3-arrays/properties and methods of arrays.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// Some properties and methods of array are--
```
Comment (sabderemane) [inline 3-arrays/properties and methods of arrays.js:1]: You should rename the file with no spaces please : `array-properties.js`

File: 3-arrays/properties and methods of arrays.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+// 1. Returns the number of elements-
```
Comment (sabderemane) [inline 3-arrays/properties and methods of arrays.js:4]:
```suggestion
// 1. Returns the number of elements :
```

File: 3-arrays/properties and methods of arrays.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+		//length
+
+//example--
```
Comment (sabderemane) [inline 3-arrays/properties and methods of arrays.js:8]:
```suggestion
// example
```

File: 3-arrays/properties and methods of arrays.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+fruits.length=2;
+
+
+
+
+//2. Sorts the array-
+
+		//sort();
+
+//example--
```
Comment (sabderemane) [inline 3-arrays/properties and methods of arrays.js:21]:
```suggestion
// 2. Sorts the array :

		//sort();

// example
```

File: 3-arrays/properties and methods of arrays.js (hunk 1 above)
Comment (sabderemane) [inline 3-arrays/properties and methods of arrays.js:1]:
```suggestion
// Some properties and methods of array are :
```
You should explain what is a property and a method, the syntax of them

File: 3-arrays/properties and methods of arrays.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+fruits.sort();
+
+
+
+
+// 3. Joins arrays and returns an array with the joined arrays-
+
+		//concat();
+
+//example--
```
Comment (sabderemane) [inline 3-arrays/properties and methods of arrays.js:34]:
```suggestion
// 3. Joins arrays and returns an array with the joined arrays :

		//concat();

// example
```

File: 3-arrays/properties and methods of arrays.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+fruits.concat();
+
+
+
+
+// 4. Joins all elements of an array into a string-
+
+		//join();
+
+//example--
```
Comment (sabderemane) [inline 3-arrays/properties and methods of arrays.js:47]:
```suggestion
// 4. Joins all elements of an array into a string :

		//join();

// example
```

File: 3-arrays/properties and methods of arrays.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+fruits.join();
+
+
+
+
+// 5. Search the array for an element and returns its position-
+
+		//indexOf();
+
+//example--
```
Comment (sabderemane) [inline 3-arrays/properties and methods of arrays.js:60]:
```suggestion
// 5. Search the array for an element and returns its position :

		//indexOf();

// example
```

**Review by sabderemane — CHANGES_REQUESTED** [review]

**Comment by sabderemane** [comment]
Thank you for your contribution @rishabh1911007, I have made some suggestions :)

**Review by rishabh1911007 — COMMENTED** [review]
Please Just add Hacktoberfest-Accepted Label.

**Review by rishabh1911007 — COMMENTED** [review]
Please just add Hacktoberfest-accepted label.

**Comment by rishabh1911007** [comment]
Now, please add Hacktoberfest-Accepted Label.

**Review by sabderemane — CHANGES_REQUESTED** [review]
It's missing a challenge as it is said in the CONTRIBUTING.md file. Please review your contribution or it will not be merged.

**Review by rishabh1911007 — COMMENTED** [review]
Now is it okay ?

**Review by rishabh1911007 — COMMENTED** [review]
Now is it okay ?

**Comment by rishabh1911007** [comment]
Now is it okay, please see it and please do add label of
Hacktoberfest2022-Accepted

File: 3-arrays/properties_and_methods_of_arrays.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+// 1. Returns the number of elements :
+
+		//length
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.length=3;
```
Comment (sabderemane) [inline 3-arrays/properties_and_methods_of_arrays.js:12]:
```suggestion
console.log(fruits.length); // output : 3
```

File: 3-arrays/properties_and_methods_of_arrays.js [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+// 2. Sorts the array :
+
+		//sort();
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.sort();
```
Comment (sabderemane) [inline 3-arrays/properties_and_methods_of_arrays.js:25]: You should add the output of the sort to show the difference

File: 3-arrays/properties_and_methods_of_arrays.js [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+
+// 3. Joins arrays and returns an array with the joined arrays :
+
+		//concat();
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.concat();
```
Comment (sabderemane) [inline 3-arrays/properties_and_methods_of_arrays.js:38]: You are doing a concat with nothing, it's not helpful for someone who is learning

File: 3-arrays/properties_and_methods_of_arrays.js [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+
+// 4. Joins all elements of an array into a string :
+
+		//join();
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.join();
```
Comment (sabderemane) [inline 3-arrays/properties_and_methods_of_arrays.js:51]: You are using the join method with nothing, not helpful for the learner. You should add the output also.

File: 3-arrays/properties_and_methods_of_arrays.js [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+
+// 5. Search the array for an element and returns its position :
+
+		//indexOf();
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.indexOf();
```
Comment (sabderemane) [inline 3-arrays/properties_and_methods_of_arrays.js:64]: You didn't put a specific index and no output also for the learner it's useless

**Review by sabderemane — CHANGES_REQUESTED** [review]
I'm sorry but it's not ok. Please stop putting pressure on the reviewers it will not make the review go faster

**Review by rishabh1911007 — COMMENTED** [review]
Updated file of properties and method of arrays.

**Review by sabderemane — APPROVED** [review]

**Comment by sabderemane** [comment]
Thank you for your contribution @rishabh1911007 ! 🔥

**Pull request merged**